### DISCUSSION TITLE
No luck selecting a Kernel in vscode

### DISCUSSION THREAD

**Discussion by MykeDu** [giswqs/geog-312#81] [Q&A] [answered]
When opening .ipynb in vscode, I cannot select a kernel.  
I only see a constantly spinning spinner Detecting Kernels.
When I try to choose a Python Environment, Detecting Kernel spins.
Confirmed ipykernel package installed in venv.
Installed all the appropriate vscode extensions

![kernel](https://github.com/user-attachments/assets/88c67968-0aff-4793-a31d-3e284cb9493f)

**Comment by giswqs** [comment, marked as answer]
See https://github.com/microsoft/vscode-jupyter/issues/15632

**Reply by MykeDu** [reply]
I ran across this thread last weekend.  It was closed because no debug log was provided.
I also uninstalled and reinstalled vscode over the weekend and tried using another laptop.
Still no luck.  Just using the online version.  Found other posts to downgrade vscode.

**Comment by giswqs** [comment]
Try this one: https://youtu.be/nMyxL4n2nI0
